api/app/tickers/tickers.py

```diff
@@ -1,11 +1,36 @@
-from fastapi import APIRouter
+from sqlalchemy.orm import Session
+import fastapi
 
-router = APIRouter(
+from ..db import crud, models, schemas
+from ..db.database import SessionLocal, engine
+from .operations import save_finance_data
+
+
+def get_db():
+    models.Base.metadata.create_all(bind=engine)
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+router = fastapi.APIRouter(
     prefix='/tickers',
     tags=['tickers'],
 )
 
 
-@router.get('', tags=['tickers'])
+@router.get("/", tags=["tickers"])
 async def get_all_tickers():
-    return [{'name': 'ticker1'}, {'name': 'ticker2'}]
+    return [{"name": "ticker1"}, {"name": "ticker2"}]
+
+
+@router.post("/stock/", response_model=schemas.Stock)
```

Comment:
this is a little confusing - the function name is `getStock`, but the endpoint is a `POST` endpoint